doc/MediaSigning.xml

```diff
@@ -914,7 +914,7 @@
         <itemizedlist>
           <listitem>
             <para>Tag version (1 byte)</para>
-            <para>Version = 1</para>
+            <para>Version = 1-2</para>
```

Comment:
Should we only keep the latest tag version (2) or all valid ones (1, 2)?

And does this change require an update of spec version from v1.0 to v1.1?